tests/test_helpers.py

```diff
@@ -115,6 +115,49 @@
      "Please rerun the helper specifying '-n list-index' to update item number 'list-index':\n"
      "1. Maria as a new contact\n"
      ),
+    (["helper", "lister", "people"],
+     "sbillinge    \nscopatz\n"),
+    (["helper", "lister", "projecta"],
+     "20ly_newprojectum    \n20sb_firstprojectum\n"),
+    (["helper", "lister", "contacts"],
+     "afriend\n"),
+    (["helper", "lister", "proposals"],
```

Comment:
I think these tests are testing the same functionality so after the first one they can be removed.  You don't have to iterate through all the collections....

Reply:
Done!